src/sentry/notifications/helpers.py

```diff
@@ -435,3 +425,37 @@ def get_reason_context(extra_context: Mapping[str, Any]) -> MutableMapping[str,
     return {
         "reason": GroupSubscriptionReason.descriptions.get(reason, "are subscribed to this issue")
     }
+
+
+def get_highest_notification_setting_value(
+    notification_settings_by_provider: Mapping[ExternalProviders, NotificationSettingOptionValues],
+) -> Optional[NotificationSettingOptionValues]:
+    """
+    Find the "most specific" notification setting value. Currently non-NEVER
```

Comment:
Is this docstring correct? I'm confused that it says it's the most specific setting, but then there's also a helper method called `get_most_specific_notification_setting_value`

Reply:
`get_highest_notification_setting_value` is a helper function that supports `get_most_specific_notification_setting_value`. Would it help if I moved it inside the other function?

Reply:
Maybe the docstring here can explain better because I find to to be worded in an unclear way. As I understand it after we talked about it, "most specific" here means "the largest `NotificationSettingOptionValues` enum value".